app/views/editorial/submissions/node_types/_news_article.html.haml

```diff
@@ -0,0 +1,3 @@
+
+%dt Release date
+%dd= content["date"]
```

Comment:
You're correct - this is wrong.

However, it means that I cannot show the release date for a node here because it's not part of the revision. I think this is highlighting the fact that we have an arbitrary mix of versioned and non-versioned fields.

I vote that for now we just delete the content of this file, while we have a rethink of how we tackle versioning.